src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-api-gateway-v2-monitoring-integration.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Amazon API Gateway V2 Monitoring Integration
+tags:
+  - Integrations
+  - Amazon integrations
+  - AWS integrations list
+metaDescription: "New Relic's AWS API Gateway v2 monitoring integration: what data it reports, and how to enable it."
+freshnessValidatedDate: 2023-07-10
```

Comment:
@rahul188 should this have a freshness date that is something that is more up to date like this: 2024-01-29?

Reply:
@rhetoric101, I made the suggested change.